test/packs/fake.ts

```diff
@@ -157,7 +158,7 @@ export const manifest: PackDefinition = createFakePack({
       identityName: 'Person',
       schema: fakePersonSchema,
       formula: {
-        name: 'Students',
+        name: 'SyncStudents',
```

Comment:
This fake pack used the same name for the sync table and its formula, so that covered up the bug.